include/cctz/civil_time_detail.h

```diff
@@ -566,6 +567,198 @@ CONSTEXPR_F civil_day prev_weekday(civil_day cd, weekday wd) noexcept {
   }
 }
 
+////////////////////////////////////////////////////////////////////////
+
+enum class weekstart
+{
+  monday_zero,
+  monday_one,
+  sunday_zero,
+  sunday_one,
+};
+
+CONSTEXPR_F int weekday_to_number(weekday wd, weekstart ws) noexcept {
+  switch(ws) {
+    case weekstart::monday_zero:
+      switch (wd) {
+        case weekday::monday:
+          return 0;
+        case weekday::tuesday:
+          return 1;
+        case weekday::wednesday:
+          return 2;
+        case weekday::thursday:
+          return 3;
+        case weekday::friday:
+          return 4;
+        case weekday::saturday:
+          return 5;
+        case weekday::sunday:
+          return 6;
+      }
+      break;
+    case weekstart::monday_one:
+      switch (wd) {
+        case weekday::monday:
+          return 1;
+        case weekday::tuesday:
+          return 2;
+        case weekday::wednesday:
+          return 3;
+        case weekday::thursday:
+          return 4;
+        case weekday::friday:
+          return 5;
+        case weekday::saturday:
+          return 6;
+        case weekday::sunday:
+          return 7;
+      }
+      break;
+    case weekstart::sunday_zero:
+      switch (wd) {
+        case weekday::sunday:
+          return 0;
+        case weekday::monday:
+          return 1;
+        case weekday::tuesday:
+          return 2;
+        case weekday::wednesday:
+          return 3;
+        case weekday::thursday:
+          return 4;
+        case weekday::friday:
+          return 5;
+        case weekday::saturday:
+          return 6;
+      }
+      break;
+    case weekstart::sunday_one:
+      switch (wd) {
+        case weekday::sunday:
+          return 1;
+        case weekday::monday:
+          return 2;
+        case weekday::tuesday:
+          return 3;
+        case weekday::wednesday:
+          return 4;
+        case weekday::thursday:
+          return 5;
+        case weekday::friday:
+          return 6;
+        case weekday::saturday:
+          return 7;
+      }
+      break;
+  }
+
+  // Undefined behavior.
+  return 0;
+}
+
+CONSTEXPR_F weekday number_to_weekday(int wd, weekstart ws, std::string* err = nullptr) noexcept {
```

Comment:
is std::string good fit? I tried to copy interface from parse(). Basically bool would
be enogh. I first also did that we return error and weekday is pointer but that did
not feel right as many times err is optional and programmer really knows already
that error cannot happend.